sshd-common/src/main/java/org/apache/sshd/common/util/GenericUtils.java

```diff
@@ -196,6 +196,49 @@ public static boolean isNotEmpty(CharSequence cs) {
         return !isEmpty(cs);
     }
 
+    /**
+     * <p>
+     * Checks if a CharSequence is empty (""), null or whitespace only.
+     * </p>
+     *
+     * <p>
+     * Whitespace is defined by {@link Character#isWhitespace(char)}.
+     * </p>
+     *
+     * <pre>
+     * StringUtils.isBlank(null)      = true
+     * StringUtils.isBlank("")        = true
+     * StringUtils.isBlank(" ")       = true
+     * StringUtils.isBlank("bob")     = false
+     * StringUtils.isBlank("  bob  ") = false
+     * </pre>
+     *
+     * @param  cs the CharSequence to check, may be null
+     * @return    {@code true} if the CharSequence is null, empty or whitespace only
+     * @since     2.0
+     * @since     3.0 Changed signature from isBlank(String) to isBlank(CharSequence)
```

Comment:
So this was copied from somewhere. Looks like StringUtils from org.apache.commons.lang, so license-wise OK.

I'll remove the misleading "since" tags and fix the comment.